pkg/verification/utils.go

```diff
@@ -26,16 +26,34 @@
 package verification
 
 import (
+	"crypto/ecdsa"
 	"crypto/hmac"
+	"crypto/rand"
 	"crypto/sha256"
+	"encoding/base64"
 	"fmt"
 	"sort"
 	"strings"
+	"testing"
+	"time"
 
+	"github.com/dgrijalva/jwt-go"
+	vm "github.com/google/exposure-notifications-server/internal/verification/model"
 	verifyapi "github.com/google/exposure-notifications-server/pkg/api/v1"
 	"github.com/google/exposure-notifications-server/pkg/api/v1alpha1"
 )
 
+// JwtConfig stores the config used to fetch a verification jwt certificate
+type JwtConfig struct {
```

Comment:
this shouldn't be in PKG since it requires that you pass in things from internal. 

needs to live in internal somewhere

Reply:
Done

Reply:
the move looks good, but this should be removed - I think just revert this file.

Reply:
Done.